OpenSim/Common/Property.h

```diff
@@ -529,6 +529,12 @@ class Property : public AbstractProperty {
         return -1;
     }
 
+    /** Return index of passed in name if the Property contains objects that are
+    derived from OpenSim::Object, and -1 if no such Object is found. Throws an 
+    Exception if the List doesn't contain OpenSim Objects (e.g. Primitive types)
+    since these are not named **/
```

Comment:
This comment should mention the time complexity -- it is useful information in some cases.
Also, there is a missing period.